Hazel/src/Hazel/Scene/Components.h

```diff
@@ -120,6 +120,7 @@ namespace Hazel {
 	{
 		glm::vec2 Offset = { 0.0f, 0.0f };
 		glm::vec2 Size = { 0.5f, 0.5f };
+		float Rotation = 0.0f;
```

Comment:
Why has this Rotation variable been added to BoxCollider2DComponent? It doesn't seem to have any UI for setting it anywhere, so it seems like it would just always be 0.

Reply:
Remove this.

Reply:
I would think a box collider should be able to be rotated independently. Instead of removing rotation. You can add the field in SceneHierarchyPanel and make it serializable.